oak/server/oak_node.cc

```diff
@@ -318,6 +320,13 @@ grpc::Status OakNode::ProcessModuleInvocation(grpc::GenericServerContext* contex
   channels_[LOGGING_CHANNEL_HANDLE] = std::move(logging_channel);
   LOG(INFO) << "Created logging channel";
 
+  // TODO: Move to initialization method.
```

Comment:
I expect this to end up being a new type of (pseudo-)node: https://github.com/project-oak/oak/blob/4523eedaf4613c821a0e8a5042ae5404d45b2fe2/oak/proto/manager.proto#L38-L42